### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#274] [General]
When I am running the Tensile_test example, this error is showing:

line search unable to reduce residual, attempting to increase residual by 10% in hopes of escaping local minimum--
  --selecting alpha = 0.25, increasing residual by 0.1--

Please help me to resolve this issue.

**Comment by ppandit95** [comment]
Hi Nikhil23121998

This is not a issue per say as it has been hard coded in peridigm to look for ways to reach local minima and the reason I see for this convergence bottleneck could be that you have applied quite large a force or displacement or else you could have taken less number of loadsteps if the boundary condition is dependent on loadsteps(i.e. time variable) in quasi static simulation.

Hope this helps.

Regards
Pushkar
